budgetportal/migrations/0043_auto_20200310_1720.py

```diff
@@ -0,0 +1,97 @@
+# Generated by Django 2.2.10 on 2020-03-10 17:20
+
+import autoslug.fields
+from django.db import migrations, models
+import django.db.models.deletion
+import wagtail.core.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wagtailimages', '0001_squashed_0021'),
+        ('wagtailcore', '0045_assign_unlock_grouppagepermission'),
+        ('budgetportal', '0042_auto_20200131_1500'),
+    ]
+
+    operations = [
+        migrations.AlterField(
```

Comment:
List of unapplied migrations - I believe someone just made it directly on DB (sql) or changed things like help text.
Easy to remove in case its not good, but its a bit annoying to remove it everytime